common/components/WalletDecrypt/PrivateKey.jsx

```diff
@@ -80,7 +84,9 @@ export default class PrivateKeyDecrypt extends Component {
 
   onPasswordChange = (e: SyntheticInputEvent) => {
     const fixedPkey = fixPkey(this.props.value.key);
-    const isValid = isValidPrivKey(fixedPkey.length);
+    const isValid =
+      isValidPrivKey(fixedPkey.length) ||
+      isValidEncryptedPrivKey(fixedPkey.length);
```

Comment:
This fix, valid, length check series of calls should be a function. It happens 3 times in here.